test/MemeBaseLaunchCampaign.js

```diff
@@ -17,6 +17,7 @@ const main = async () => {
     const defaultDeposit = ethers.utils.parseEther("1500");
     const oneDay = 86400;
     const twoDays = 2 * oneDay;
+    const gasLimit = 10000000;
```

Comment:
This is needed as sometimes tx-s fail due to miscalculation of gas cost, this is evident on VNet debugging.